CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+#### 0.12.0+4
+  * Support package names with hyphens (even though it violates dart style).
```

Comment:
Well, people rarely publish their applications to pub, so any restrictions it has don't really apply? I don't think its reasonable for the web components transformer to be the place where this fails.